easybuild/easyconfigs/m/matplotlib/matplotlib-1.5.1-foss-2016a-Python-2.7.11-freetype-2.6.3.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '1.5.1'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+freetype_ver = '2.6.3'
+versionsuffix = '-Python-%%(pyver)s-freetype-%s' % freetype_ver
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('freetype', freetype_ver),
+    ('libpng', '1.6.21'),
+]
+
+exts_list = [
+    ('Cycler', '0.9.0', {
+        'modulename': 'cycler',
+        'source_urls': ['https://pypi.python.org/packages/source/C/Cycler'],
+        'source_tmpl': 'cycler-%(version)s.tar.gz',
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib'],
```

Comment:
please include this line below, which fixes a problem (segfault) with linking `matplotlib` to the EB-provided Tcl/Tk:

```python
'patches': ['matplotlib-1.5.1_fix-Tcl-Tk-libdir.patch']
```

cfr. #2971